### PR TITLE
US011-Create-Login-Form

### PR DESCRIPTION
**Pull request by DuongPhamViet**

### CODE REVIEW

File: B-CLIENT/gotrip/src/App.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import logo from './logo.svg';
 import './App.css';
+import './login.scss';
 import { BrowserRouter, Switch, Route, Link, Router } from 'react-router-dom';
 import AdminPage from './admin-page/AdminPage';
-
+import login from './auth/login/login';
+import register from './auth/registration/register';
+import resetpass from './auth/login/resetpass';
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/App.js:8]: Change to reset(reset)

File: B-CLIENT/gotrip/src/auth/registration/register.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const register = () => {
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/auth/registration/register.jsx:1]: Remove all register feature;tránh conflict với @chungdaotn

**Review by dnhieu1992 — CHANGES_REQUESTED** [review]

File: B-CLIENT/gotrip/src/App.js [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import logo from './logo.svg';
 import './App.css';
+import './login.scss';
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/App.js:3]: Add to index.scss file

File: B-CLIENT/gotrip/src/App.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
...
 import { BrowserRouter, Switch, Route, Link, Router } from 'react-router-dom';
 import AdminPage from './admin-page/AdminPage';
-
+import login from './auth/login/login';
+import reset from './auth/login/reset';
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/App.js:7]: remove reset component, change login name component to Login, name of the component is PascalCase

File: B-CLIENT/gotrip/src/App.js [hunk 5]
```diff
@@ -11,6 +13,8 @@ function App() {
         {/* <Route exact path="/" component={LoginContainer} exact/> */}
         <Route path="/" exact component={AdminPage} />
         <Route path="/admin" component={AdminPage} />
+        <Route path="/login" component={login} />
+        <Route path="/login/reset" component={reset} />
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/App.js:17]: Remove reset route

File: B-CLIENT/gotrip/src/admin-page/bed/component/BedForm.jsx [hunk 6]
```diff
@@ -63,7 +63,8 @@ const BedForm = ({
                         value={description}
                         required
                         validationError={BED_TEXT_CONFIG.REQUIRED_FIELD_MSG}
-                    />
+                    >
+                    </FormsyElement.FormsyTextarea>
                     {/* <label>Description</label>
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/admin-page/bed/component/BedForm.jsx:68]: remove code not use

File: B-CLIENT/gotrip/src/auth/login/reset.jsx [hunk 7]
```diff
@@ -0,0 +1,4 @@
+const reset = () => {
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/auth/login/reset.jsx:1]: Component name is PascalCase

File: B-CLIENT/gotrip/src/index.scss [hunk 8]
```diff
@@ -1,5 +1,5 @@
 @import "~bootstrap/scss/bootstrap.scss";
 @import "~bootstrap/scss/bootstrap-grid.scss";
-@import "./shared/sass/login/login.scss";
+@import "./login.scss";
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/index.scss:3]: move login.scss to /shared/sass/components/login

File: B-CLIENT/gotrip/src/App.js (hunk 3 above)
Comment (DuongPhamViet) [reply B-CLIENT/gotrip/src/App.js]: done

File: B-CLIENT/gotrip/src/App.js (hunk 4 above)
Comment (DuongPhamViet) [reply B-CLIENT/gotrip/src/App.js]: done

File: B-CLIENT/gotrip/src/App.js (hunk 5 above)
Comment (DuongPhamViet) [reply B-CLIENT/gotrip/src/App.js]: done

File: B-CLIENT/gotrip/src/admin-page/bed/component/BedForm.jsx (hunk 6 above)
Comment (DuongPhamViet) [reply B-CLIENT/gotrip/src/admin-page/bed/component/BedForm.jsx]: done

File: B-CLIENT/gotrip/src/auth/login/reset.jsx (hunk 7 above)
Comment (DuongPhamViet) [reply B-CLIENT/gotrip/src/auth/login/reset.jsx]: done

File: B-CLIENT/gotrip/src/index.scss (hunk 8 above)
Comment (DuongPhamViet) [reply B-CLIENT/gotrip/src/index.scss]: done

File: B-CLIENT/gotrip/src/auth/login/login.jsx [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+
+import {
+    loginUser
+} from './api/apiHandle.js';
+
+const {
+    FormsyInput
+} = FormsyElement
+
+const Login = () => {
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/auth/login/login.jsx:16]: name of file same with name of component(PasscalCase)

File: B-CLIENT/gotrip/src/auth/login/reset.jsx [hunk 10]
```diff
@@ -0,0 +1,4 @@
+const Reset = () => {
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/auth/login/reset.jsx:1]: Name of file same with name of component(PasscalCase)

Comment (DuongPhamViet) [reply B-CLIENT/gotrip/src/auth/login/reset.jsx:1]: done

File: B-CLIENT/gotrip/src/auth/login/login.jsx (hunk 9 above)
Comment (DuongPhamViet) [reply B-CLIENT/gotrip/src/auth/login/login.jsx:16]: done

**Pull request merged**